### DISCUSSION TITLE
How to link against the OS libc++

### DISCUSSION THREAD

**Discussion by jpsalm** [android/ndk#1943] [Q&A] [answered]
We have an AOSP project in which we would like to integrate a third party library which itself is a large cmake project.

Rather than rewrite the cmake, I'm just investigating if we can build the project with the ndk.  The project builds fine but as the consumer of this library needs access to libutils/libcutils and the full libbinder using libc++_shared isn't really an option.  Is there a way to link against libc++ directly?  I don't see an option in the NDK's toolchain.cmake and the make_standalone_toolchain.py doesn't seem to allow for specifying an STL anymore  (on r23c at least).

**Comment by DanAlbert** [comment, marked as answer]
You can't. The ABI is not stable.
